addons/web/static/src/views/kanban/kanban_renderer.js

```diff
@@ -447,13 +447,18 @@ export class KanbanRenderer extends Component {
         });
     }
 
+    getDropdownContainer(groupId) {
```

Comment:
I'd rename this. `getDropdownContainer` tells what this function is used for (single use for now), instead of what it does. If (even though it's unlikely) this function is called somewhere else for another need, it will be weird to ask for the "dropdown container". So for instance `getGroupEl`.